examples/tensorflow-tabular/sim.py

```diff
@@ -0,0 +1,288 @@
+import os
+import argparse
+from typing import Dict, List, Tuple
+
+import tensorflow as tf
+
+import flwr as fl
+from flwr.common import Metrics
+from flwr.simulation.ray_transport.utils import enable_tf_gpu_growth
+
+from datasets import Dataset, concatenate_datasets
+from flwr_datasets import FederatedDataset
+from flwr_datasets.partitioner import DirichletPartitioner
+
+import numpy as np
+import pandas as pd
+
+from datasets import load_dataset
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+
+# Make TensorFlow logs less verbose
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+parser = argparse.ArgumentParser(description="Flower Simulation with Tensorflow/Keras for tabular dataset")
+
+parser.add_argument(
+    "--num_cpus",
+    type=int,
+    default=1,
+    help="Number of CPUs to assign to a virtual client",
+)
+parser.add_argument(
+    "--num_gpus",
+    type=float,
+    default=0.0,
+    help="Ratio of GPU memory to assign to a virtual client",
+)
```

Comment:
Based on my comment about having this in "FlowerNext"-only mode below, let's remove it?